cert-walletTests/CertificateParserTests.swift

```diff
@@ -26,6 +27,18 @@ class CertificateParserTests: XCTestCase {
         XCTAssertEqual(certificate?.version, .oneDotOne)
     }
     
+    func testExpectingV1_1SignedCertificate() {
+        let testBundle = Bundle(for: type(of: self))
+        guard let fileUrl = testBundle.url(forResource: v1_1signedFilename, withExtension: "json") ,
+            let file = try? Data(contentsOf: fileUrl) else {
+                return
+        }
+        
+        let certificate = CertificateParser.parse(data: file)
+        XCTAssertNotNil(certificate)
+        XCTAssertEqual(certificate?.version, .oneDotOne)
```

Comment:
`.oneDotOne` is such a regretful name. I can't imagine having to read this code to someone over the phone.